packages/ckeditor5-dev-release-tools/lib/release/updateversions.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { globSync } = require( 'glob' );
+const path = require( 'path' );
+const fs = require( 'fs-extra' );
+const semver = require( 'semver' );
+const getPackageJson = require( '../utils/getpackagejson' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
```

Comment:
Just the order: external imports are first, then the local ones.

```suggestion
const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
const getPackageJson = require( '../utils/getpackagejson' );
```